azure-pipelines/end-to-end-tests-dir/ci.ps1

```diff
@@ -69,20 +69,21 @@ $Output = Run-VcpkgAndCaptureOutput ci @commonArgs --x-builtin-ports-root="$empt
 Restore-Problem-Matchers
 Throw-IfNotFailed
 
-# test effect of parent hashes
-Remove-Item -Recurse -Force $installRoot -ErrorAction SilentlyContinue
-New-Item -ItemType Directory -Path $installRoot -Force | Out-Null
-Remove-Item -Recurse -Force $ArchiveRoot -ErrorAction SilentlyContinue
-New-Item -ItemType Directory -Path $ArchiveRoot -Force | Out-Null
-# Not a dry run in order to populate the artifact cache.
-$Output = Run-VcpkgAndCaptureOutput ci @commonArgs --x-builtin-ports-root="$PSScriptRoot/../e2e-ports/ci"  --binarysource="clear;files,$ArchiveRoot" --output-hashes="$TestingRoot/parent-hashes.json"
```

Comment:
`--binarysource="clear;files,$ArchiveRoot"` didn't achieve "to populate the artifact cache": It is using the default mode which is read. :facepalm: